### PR TITLE
[MGDOBR-393]: Configure i18n and setup default EN locale dictionary

### PR DESCRIPTION
**Pull request by kelvah**
https://issues.redhat.com/browse/MGDOBR-393

I've added a translation to display the "Demo Overview" title in the main page as a way to test the i18n setup.

You can run the OB application in standalone mode following the instructions in the [README](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui#readme).

To run the OB federated module while we still don't have proper `app-services-ui` integration, you can follow this steps:
- first run `sandbox-ui` in federated mode using `npm run start:federate`
- clone the `OB-route` branch of [my fork of app-services-ui](https://github.com/kelvah/app-services-ui/tree/OB-route)
- run `npm install` and then `npm start:dev` in the `app-services-ui` app
- open the following url in the browser: https://prod.foo.redhat.com:1337/beta/application-services/openbridge

### CODE REVIEW

**Review by manstis — APPROVED** [review]
LGTM :+1:

**Review by vpellegrino — COMMENTED** [review]
Just one minor thing, nothing blocking

**Review by vpellegrino — APPROVED** [review]
LGTM, thank you @kelvah

File: src/App.tsx
Comment (kelvah) [reply src/App.tsx:25]: > Do/will we use Relay? https://reactjs.org/docs/concurrent-mode-suspense.html#what-if-i-dont-use-relay

Not that I'm aware of. In this case suspense is there to wait for i18n to get initialized and fetch/import dictionaries.

File: src/app/OBInstance/OBInstancesListPage/OBInstancesListPage.tsx
Comment (manstis) [reply src/app/OBInstance/OBInstancesListPage/OBInstancesListPage.tsx:95]: Don't be hasty.. now I know `t` I will not forget it... 

...and you're right, a longer name could become unwanted clutter as the UI components become more complicated.

I've approved the PR with `t` so unless @vpellegrino really wants the change I'm OK.

Comment (vpellegrino) [reply src/app/OBInstance/OBInstancesListPage/OBInstancesListPage.tsx:95]: @manstis @kelvah, `t` is fine for me too. Indeed I approved w/o blocking this PR.

Comment (kelvah) [reply src/app/OBInstance/OBInstancesListPage/OBInstancesListPage.tsx:95]: :ok_hand:

File: package.json
Comment (jomarko) [inline package.json:20]: https://www.npmjs.com/package/@rhoas/app-services-ui-components?activeTab=readme says it is not recommended install as direct dependency. Do we meet this recommendation? Sorry if that is a silly question. I have more experience with testing by cypress than development in react. 

Also the latest seems to be a little bit higher `1.24.1`. Is the current version expected?

**Review by jomarko — COMMENTED** [review]
Hello team. Thank you for this PR. 

The app loaded correctly for me in the first case - standalone mode. However the federated module mode seems to result in error state on my machine.

Screenshot from 2022-03-15 10-54-26

File: package.json
Comment (kelvah) [reply package.json:20]: > https://www.npmjs.com/package/@rhoas/app-services-ui-components?activeTab=readme says it is not recommended install as direct dependency. Do we meet this recommendation? Sorry if that is a silly question. I have more experience with testing by cypress than development in react.

It's not a silly question at all. Actually it's an important aspect.
I think the recommendation you linked is not necessary.
By declaring dependencies as shared ([here](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/blob/main/webpack.common.js#L130)), they are loaded at runtime only if the host application hasn't already done it.
In this way the same dependency is loaded only once by the first module that requests it.
Given that, I'd say it's not necessary to rely to `peerDependencies`. If the shared dep version matches an existing dep of the host application, that's enough.

> 
> Also the latest seems to be a little bit higher `1.24.1`. Is the current version expected?

I'm aligning with other repos here. They are all using this version of PF. So if we want to reuse it without further dependency loading, we have to stick to that. 
We could argue if it make sense to use fixed version numbers instead of `^version`, but that's another discussion :)

**Pull request merged**
